### PR TITLE
fix not to emit version

### PR DESCRIPTION
**Pull request by rchaser53**
related: https://github.com/rust-lang/rustfmt/issues/3505

### CODE REVIEW

File: src/cargo-fmt/main.rs [hunk 1]
```diff
@@ -122,8 +122,19 @@ fn handle_command_status(status: Result<i32, io::Error>, opts: &getopts::Options
...
 }
 
-fn get_version(verbosity: Verbosity) -> Result<i32, io::Error> {
-    run_rustfmt(&BTreeSet::new(), &[String::from("--version")], verbosity)
+fn get_version() -> Result<i32, io::Error> {
+    let mut status = vec![];
+    let mut command = Command::new("rustfmt")
+        .stdout(std::process::Stdio::inherit())
+        .args(&[String::from("--version")])
+        .spawn()?;
```
Comment (scampi) [inline src/cargo-fmt/main.rs]: Can you add the error mapping ?

https://github.com/rust-lang/rustfmt/pull/3506/files#diff-0f04fceb10cfdeda3716c7824a8dc2deL361

Comment (rchaser53) [reply src/cargo-fmt/main.rs]: Thank you the review.
I added the error mapping.

File: src/cargo-fmt/main.rs [hunk 2]
```diff
@@ -122,8 +122,26 @@ fn handle_command_status(status: Result<i32, io::Error>, opts: &getopts::Options
     }
 }
 
-fn get_version(verbosity: Verbosity) -> Result<i32, io::Error> {
-    run_rustfmt(&BTreeSet::new(), &[String::from("--version")], verbosity)
+fn get_version() -> Result<i32, io::Error> {
+    let mut status = vec![];
```
Comment (topecongiro) [inline src/cargo-fmt/main.rs]: Since we only run `rustfmt` once, we need not use a vector to keep an exit status.

Comment (rchaser53) [reply src/cargo-fmt/main.rs]: Thanks review. I avoided a vector.

**Comment by scampi** [comment]
build failures are unrelated to this PR

**Pull request merged**